packages/calypso-products/src/features-list.tsx

```diff
@@ -2767,4 +2784,88 @@ export const FEATURES_LIST: FeatureList = {
 		getTitle: () => i18n.translate( 'Advanced Jetpack features' ),
 	},
 	/* END: Sensei Features */
+
+	/* Start: Trail Map new Features */
+	[ FEATURE_PRE_INSTALLED_ECOMMERCE_PLUGINS ]: {
+		getSlug: () => FEATURE_PRE_INSTALLED_ECOMMERCE_PLUGINS,
+		getTitle: () => i18n.translate( '25+ pre-installed ecommerce plugins ($1,500/year value)' ),
+	},
+	[ FEATURE_20_PREMIUM_THEMES ]: {
+		getSlug: () => FEATURE_20_PREMIUM_THEMES,
+		getTitle: () => i18n.translate( '20 premium themes' ),
+	},
+	[ FEATURE_48_PREMIUM_THEMES ]: {
+		getSlug: () => FEATURE_48_PREMIUM_THEMES,
+		getTitle: () => i18n.translate( '48 premium themes' ),
+	},
+	[ FEATURE_AD_SUPPORTED_EXPERIENCE ]: {
+		getSlug: () => FEATURE_AD_SUPPORTED_EXPERIENCE,
+		getTitle: () => i18n.translate( 'Ad-supported experience' ),
+	},
+	[ FEATURE_REAL_TIME_STATS ]: {
+		getSlug: () => FEATURE_REAL_TIME_STATS,
+		getTitle: () => i18n.translate( 'Advanced Real Time stats' ),
+	},
+	[ FEATURE_AUTOMATED_BURST_SCALING ]: {
+		getSlug: () => FEATURE_AUTOMATED_BURST_SCALING,
+		getTitle: () => i18n.translate( 'Automated burst scaling' ),
+	},
+	[ FEATURE_DATABASE_ACCESS ]: {
+		getSlug: () => FEATURE_DATABASE_ACCESS,
+		getTitle: () => i18n.translate( 'Database Access' ),
```

Comment:
"Access" doesn't need to be capitalized at the initial. In a follow-up PR, let's sneak in an update of this from "Database Access" to "Database access"